samples/nrf9160/lte_ble_gateway/src/main.c

```diff
@@ -7,19 +7,24 @@
 #include <zephyr.h>
 #include <stdio.h>
 #include <string.h>
-#include <drivers/gps.h>
+#include <nrf_modem_at.h>
+#include <nrf_modem_gnss.h>
 #include <drivers/sensor.h>
 #include <console/console.h>
 #include <net/nrf_cloud.h>
+#include <net/nrf_cloud_agps.h>
 #include <dk_buttons_and_leds.h>
 #include <modem/lte_lc.h>
 #include <sys/reboot.h>
 #include <modem/nrf_modem_lib.h>
+#include <logging/log.h>
 
 #include "aggregator.h"
 #include "ble.h"
 #include "alarm.h"
 
+LOG_MODULE_REGISTER(lte_ble_gw, CONFIG_LOG_DEFAULT_LEVEL);
```

Comment:
I think it would be good to have own compile time log level for the sample, otherwise the only way to change the logging level for this module is to change the default log level, which probably affects some other modules as well. For example:
```suggestion
LOG_MODULE_REGISTER(lte_ble_gw, CONFIG_LTE_BLE_GW_LOG_LEVEL);
```